.github/workflows/publish-community-api.yml

```diff
@@ -0,0 +1,40 @@
+name: Build-Push kotal community api
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set Env
+      run: |
+        echo "BUILD_SHA=${GITHUB_SHA:0:7}" >> $GITHUB_ENV
+        echo "RELEASE_TAG=$(curl -s GET https://api.github.com/repos/kotalco/kotal/releases | jq -r '.[].name' | head -n1)" >> $GITHUB_ENV
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v2
+      with:
+        username: kotalco
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+    - name: Build and push
+      uses: docker/build-push-action@v3
+      with:
+        context: .
+        platforms: linux/amd64,linux/arm64
+        push: true
+        tags: kotalco/kotal-community-api:latest,kotalco/kotal-community-api:${{ env.BUILD_SHA }},kotalco/kotal-community-api:${{ env.RELEASE_TAG }}
```

Comment:
rename image name to just `community-api`, no need for `kotal-` at the beginning.